pkg/identityserver/emails/api_key_created.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package emails
+
+import "go.thethings.network/lorawan-stack/pkg/ttnpb"
+
+// APIKeyCreated is the email that is sent when users creates a new API key
+type APIKeyCreated struct {
+	Data
+	Key *ttnpb.APIKey
+}
+
+// TemplateName returns the name of the template to use for this email.
+func (APIKeyCreated) TemplateName() string { return "api_key_created" }
+
+const apiKeyCreatedSubject = `An API key for {{.Network.Name}} has been created`
```

Comment:
Not sure the network name needs to be in here (subject too long)
Same for other emails.